addon/formatters/-base.js

```diff
@@ -25,6 +28,30 @@ var FormatBase = Ember.Object.extend({
         if (match) {
             return options;
         }
+    },
+
+    _format: function (value, options, formatOptions) {
```

Comment:
Continue to rewrite to make these "dumb".  `locales` and `formats` should be passed in rather than looked up on Intl service.